<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>application_pages</class>
 <widget class="QStackedWidget" name="application_pages">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>363</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <widget class="QWidget" name="page_1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>70</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>70</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 700 12pt &quot;Segoe UI&quot;;
color: rgb(30, 131, 14);</string>
         </property>
         <property name="text">
          <string>Usuário</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLineEdit" name="lbl_user">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	padding: 6px;
	border: 2px solid #56AB49;
	border-radius: 8px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Informe o usuário...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_login">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>36</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>36</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	width: 80px;
	font: 700 italic 10pt &quot;Segoe UI&quot;;
	background-color: #1E830E;
	color: #F3F2F2;
	border-radius: 8px;
	border: none;
	padding: 6px;
}
QPushButton:hover {
	background-color: #56AB49;
}
QPushButton:pressed {
	background-color: #A2FD94;
}</string>
           </property>
           <property name="text">
            <string>Entrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_2">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>350</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLineEdit" name="field_search">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
	padding: 6px;
	border: 2px solid #56AB49;
	border-radius: 8px;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_search">
           <property name="styleSheet">
            <string notr="true">QPushButton {
	width: 80px;
	font: 700 10pt &quot;Segoe UI&quot;;
	background-color: #1E830E;
	color: #F3F2F2;
	border-radius: 8px;
	border: none;
	padding: 6px;
}
QPushButton:hover {
	background-color: #56AB49;
}
QPushButton:pressed {
	background-color: #A2FD94;
}</string>
           </property>
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QListView" name="list_colab">
           <property name="styleSheet">
            <string notr="true">QListView {
	border-radius: 8px;
	border: 2px solid #A2FD94;
}</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="0">
            <widget class="QPushButton" name="btn_ficha">
             <property name="styleSheet">
              <string notr="true">QPushButton {
	width: 80px;
	font: 700 10pt &quot;Segoe UI&quot;;
	background-color: #1E830E;
	color: #F3F2F2;
	border-radius: 8px;
	border: none;
	padding: 6px;
}
QPushButton:hover {
	background-color: #56AB49;
}
QPushButton:pressed {
	background-color: #A2FD94;
}</string>
             </property>
             <property name="text">
              <string>Ficha Ind.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="btn_detail">
             <property name="styleSheet">
              <string notr="true">QPushButton {
	width: 80px;
	font: 700 10pt &quot;Segoe UI&quot;;
	background-color: #1E830E;
	color: #F3F2F2;
	border-radius: 8px;
	border: none;
	padding: 6px;
}
QPushButton:hover {
	background-color: #56AB49;
}
QPushButton:pressed {
	background-color: #A2FD94;
}</string>
             </property>
             <property name="text">
              <string>Detalhe</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_add">
             <property name="styleSheet">
              <string notr="true">QPushButton {
	width: 80px;
	font: 700 10pt &quot;Segoe UI&quot;;
	background-color: #1E830E;
	color: #F3F2F2;
	border-radius: 8px;
	border: none;
	padding: 6px;
}
QPushButton:hover {
	background-color: #56AB49;
}
QPushButton:pressed {
	background-color: #A2FD94;
}</string>
             </property>
             <property name="text">
              <string>Adicionar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="page_3">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>PÁGINA 03</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
